# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../chkrootkit/head.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
diff -Naur chkrootkit-0.44.orig/chkrootkit chkrootkit-0.44/chkrootkit
--- chkrootkit-0.44.orig/chkrootkit	2004-09-01 14:27:28.000000000 +0100
+++ chkrootkit-0.44/chkrootkit	2004-11-10 16:06:21.000000000 +0000
@@ -1054,7 +1054,7 @@
 getCMD() {
 
    RUNNING=`${ps} ${ps_cmd} | ${egrep} "${L_REGEXP}${1}${R_REGEXP}" | \
-            ${egrep} -v egrep | ${egrep} -v chkrootkit | ${head} -1 | \
+            ${egrep} -v egrep | ${egrep} -v chkrootkit | ${head} -n 1 | \
             ${awk} '{ print $5 }'`
 
    for i in ${ROOTDIR}${RUNNING} ${ROOTDIR}usr/sbin/${1} `loc ${1} ${1} $pth`
@@ -2178,7 +2178,7 @@
     TCPD_INFECTED_LABEL="p1r0c4|hack|/dev/xmx|/dev/hdn0|/dev/xdta|/dev/tux"
 
     [ -r ${ROOTDIR}etc/inetd.conf ] &&
-    CMD=`${egrep} '^[^#].*tcpd' ${ROOTDIR}etc/inetd.conf | ${head} -1 | \
+    CMD=`${egrep} '^[^#].*tcpd' ${ROOTDIR}etc/inetd.conf | ${head} -n 1 | \
          ${awk} '{ print $6 }'`
     if ${ps} auwx | ${egrep} xinetd | ${egrep} -v grep >/dev/null 2>&1;  then
        CMD=`loc tcpd tcpd $pth`
